### PR TITLE
use python3 also for stretch

### PR DESCRIPTION
**Pull request by s-martin**
**UNTESTED YET**, because I don't have a stretch environment

Issues #747 and #749 are probably related

### CODE REVIEW

**Comment by ZyanKLee** [comment]
in that case it might be a good idea to re-enable the python3.5 flake8 action, too.

File: scripts/installscripts/stretch-install-default.sh [hunk 1]
```diff
@@ -388,8 +388,10 @@ sudo iwconfig wlan0 power off
 
 # Install required packages
 sudo apt-get update
-sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
+sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python3-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
```
Comment (ZyanKLee) [inline scripts/installscripts/stretch-install-default.sh]: we need at least pip to be updated to python3, too. what about mutagen?
```suggestion
sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python3-dev python3-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python3-mutagen python3-gpiozero
```

File: scripts/installscripts/stretch-install-spotify.sh [hunk 2]
```diff
@@ -463,8 +463,10 @@ sudo iwconfig wlan0 power off
 
 # Install required packages
 sudo apt-get update
-sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
+sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python3-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
```
Comment (ZyanKLee) [inline scripts/installscripts/stretch-install-spotify.sh]:
```suggestion
sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python3-dev python3-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python3-mutagen python3-gpiozero
```

Comment (ZyanKLee) [reply scripts/installscripts/stretch-install-spotify.sh]: We need to keep in mind that on stretch mopidy is available only at version 2.2.3 at the most with `python:any (<< 2.8), python:any (>= 2.7.5-5~)` as python version limits.

**Comment by Vanierica** [comment]
I´ve tested this installation script on fresh installed stretch. I got several errors. Here is the output. Maybe this helps.
[Stretch-python3-on-stretch-brand.txt](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/4101763/Stretch-python3-on-stretch-brand.txt)

**Comment by MiczFlor** [comment]
Hi @s-martin without a test on stretch, it would be a bit risky to merge with develop, no?

**Comment by s-martin** [comment]
I agree, I wanted to do some more tests, when #751 is merged.

**Comment by MiczFlor** [comment]
Hi @s-martin I merged #751 and - I guess as expected - there are coflicts now with your pull request. You said "I wanted to do some more tests, when #751 is merged." Now would be the time. :)

**Comment by s-martin** [comment]
Thanks for the update.
I'll rebase the PR with develop and fix the issues.

**Comment by s-martin** [comment]
tested on docker stretch the classic installation, seems to work, but worked also previously without the change.

Unfortunetely, I don't have a test env with real hardware right now.

**Pull request merged**